### DISCUSSION TITLE
三月更新

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#787] [通知]
**📣 此版本包含大量新功能哟~强烈建议更新！😆**

## 🚀 功能

- 添加工具箱 @Richasy (#786)
    - AV/BV号转换
    - 封面下载
- 添加更多BBDown可配置项 @Richasy (#785)
    - 支持自定义下载文件夹
    - 支持下载弹幕
    - 适配 1.4.8 的更多设置
- 支持在评论中显示表情 @Richasy (#784)
- 支持在用户界面搜索视频 @Richasy (#782)
- 支持右键视频在浏览器中打开 @Richasy (#781)
- 支持固定UP主到首页 @Richasy (#780)  
    *建议不超过5个*

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2203.1.0'>三月更新</a>.</em>

**Comment by Zero-wyc** [comment]
好！

**Comment by lcx-0504** [comment]
评论区的更多内容  …   这个按钮点击之后显示的内容，有时是错误的
